### ISSUE TITLE
Hashes without brackets

### ISSUE THREAD

**Issue by B7th** [stylus/stylus#2226]
I personally find that, as it has been done elsewhere in stylus, brackets should be optional for hashes, so the syntax can look a bit more like yml/coffeescript objects.

What do you think?

**Comment by Panya** [comment]
Nope, sorry. It'd be very hard to parse (diff between blocks with declarations and hashes).

**Comment by B7th** [comment]
Would it be really that much harder to parse?
If an equal sign or a semicolon is the last element of the outdented line, I feel like that should be enough to help parsing.
Please consider as it could make importing yml like file more easily.
